### DISCUSSION TITLE
Mars 3 ( non pro) works with multiple exposure calibration times.

### DISCUSSION THREAD

**Discussion by dokipen3d** [sn4k3/UVtools#885] [General]
I did a test file (without any resin) and the mars 3 correctly exposes different parts of the screen at different exposure times without lifting the plate (well actually it lifts it a tiny amount then puts it back down very quickly).

This is going to save so much time finding the right exposure for various resins :)

**Comment by Hazoooka** [comment]
You also have Calibration > Exposure time finder, made specifically for the purpose

**Comment by dokipen3d** [comment]
Yes that what I am using :) I should have been more specific. That is what is working.
